docker/registry/manylinux2014_x86_64/gradle.properties

```diff
@@ -0,0 +1,3 @@
+io.deephaven.project.ProjectType=DOCKER_REGISTRY
+deephaven.registry.imageName=manylinux2014_x86_64:1
```

Comment:
The imageName _should_ be the name we can use to pull the image (for example, when we want to update them). I think this should be `quay.io/pypa/manylinux2014_x86_64:latest`?